mne/preprocessing/maxwell.py

```diff
@@ -588,12 +584,10 @@ def _cart_to_sph(cart_pts):
     sph_pts : ndarray, shape (n_points, 3)
         Array containing points in spherical coordinates (rad, azimuth, polar)
     """
-
     rad = np.sqrt(np.sum(cart_pts * cart_pts, axis=1))
     az = np.arctan2(cart_pts[:, 1], cart_pts[:, 0])
     pol = np.arccos(cart_pts[:, 2] / rad)
-
-    return np.c_[rad, az, pol]
```

Comment:
use of `r_` and `c_` is not recommended; they are meant to be used as shortcuts in command-line use not production code